examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
I'm not sure a flag is needed, since the class already parses all categories. The only difference is that the`categories` property doesn't list inactive ones, but you can still access them easily.

I would say that accessing inactive categories is a very rare use case - probably mostly fixing a mistake where the user laboriously defined an averaging category, then inactivated it and recorded their data.

As an example dataset, I think the phantom data is worse than the current one - it only includes one averaging category that makes sense (by accident :), and even that is inactive so you cannot get it "normally" (by the property decorator).